src/account.js

```diff
@@ -0,0 +1,23 @@
+
+/* Account login state manager
+ */
+
+import _ from 'lodash'
+import db from './db'
+import { storeKey } from './config'
+
+export default { setUser }
```

Comment:
Ok, but still, this function is pretty isolated is rather isolated and I feel like the way this is currently structure the `setUser` function should somehow operate on the `account` object that is default exported. Perhaps it should be a named export rather than the default... but we should always have a default export, so not sure what the best approach should be.